src/test/java/org/cactoos/list/JoinedTest.java

```diff
@@ -34,8 +36,9 @@
  * @since 0.20
  * @checkstyle JavadocMethodCheck (500 lines)
  * @checkstyle MagicNumber (500 line)
+ * @checkstyle DiamondOperatorCheck (500 lines)
  */
-@SuppressWarnings("PMD.TooManyMethods")
+@SuppressWarnings({"PMD.TooManyMethods", "PMD.AvoidDuplicateLiterals"})
```

Comment:
@fanifieiev Please put the values used in the tests into constants, to avoid this supression